docs/contributing/releasing-islandora.md

```diff
@@ -1,18 +1,26 @@
 # Releasing Islandora
 
-Follow these steps to release all of the Islandora ecosystem. Due to dependencies, this must be done in a particular order. While this may seem like a daunting task, remember that at any point in time you can delete/update/re-do a release in Github. If something gets botched simply
+Islandora is an ecosystem of repositories and follows [Semantic Versioning](https://semver.org/). This allows the community to remain aligned with [Drupal's approach](https://www.drupal.org/node/3108648)and support more a more modular approach and more frequent releases, as well as better upgrade paths for those using components of the system. In semantic versioning, a version has three sections 'MAJOR.MINOR.PATCH'. This looks something like 2.1.1, or you may see major versions labelled as 2.x.x. To guide repository maintainers, we recommend you increment the:
+
+* MAJOR version when you make incompatible API changes,
+* MINOR version when you add functionality in a backwards compatible manner, and
+* PATCH version when you make backwards compatible bug fixes.
+ 
+## Advice for Releasing
+
+Dependencies mean that if you are going to release all of the Islandora ecosystem, **order is very important**. At any point, rleeases can be deleted, updated, and redone in Github. You can reach out to the community if you have questions. Note that if you want to 'redo' a release, you can follow these steps:
 
 1. Delete the release in Github through their UI
 2. Delete the tag in Git both locally and remotely: `git tag -d TAG_NAME; git push --delete origin TAG_NAME`
-3. Try again.
+3. Begin Releasing again.
+
+You cannot follow these steps when publishing to Sonatype with Alpaca, but this should rarely be an issue. Version numbers can also be incremented and a new release authored.
```

Comment:
"Version numbers can also be incremented and a new release authored." -> "Instead, increment the version number and tag a new release."